modules/button/react/stories/testing/stories_IconButton.tsx

```diff
@@ -0,0 +1,59 @@
+/// <reference path="../../../../../typings.d.ts" />
+/** @jsx jsx */
+import {CSSObject, jsx} from '@emotion/core';
+import * as React from 'react';
+
+import * as systemIcons from '@workday/canvas-system-icons-web';
+import {IconButton} from '../../index';
+
+export default {
+  title: 'Testing|React/Buttons/Button/Icon Button',
+  component: IconButton,
+};
+
+const iconCellStyles: CSSObject = {
+  display: 'inline-block',
+  margin: '0 10px 10px 0',
+  textAlign: 'center',
+  width: '75px',
+
+  p: {
+    fontSize: '12px',
+    overflow: 'hidden',
+    textOverflow: 'ellipsis',
+  },
+};
+
+export const IconButtonCircleGrid = () => {
```

Comment:
I added this manual testing story (no snapshots enabled) to more easily verify the fix was working for all icons. [Link to storybook build](https://5d854c26ba934e0020f5e98a-rrnaqglbcb.chromatic.com/?path=/story/testing-react-buttons-button-icon-button--icon-button-circle-grid).

It may make sense to enable snapshots so we can catch visual regressions automatically. 🤔

Reply:
Following up with some additional thoughts @lychyi and I had on this topic.

We were thinking it would make sense to add a snapshot for this story with all buttons toggled on by default. I believe this would be helpful, but there are cases where it would be difficult to resolve visual diffs.

Each icon is an object with the following properties:
```
{
  category: "Profile",
  filename: "wd-icon-academic-appointment-title.svg",
  name: "academicAppointmentTitle",
  svg: "<svg class="wd-icon-academic-appointment-title wd-icon">...</svg>",
  tags: ["schedule", "school", "consult", "consultation", "meeting", "clipboard"],
  type: "system",
}
```

The icons in the grid are sorted alphabetically by `name`. This creates potential issues if brand new icons are added. We're unable to sort the icons in a way that guarantees new icons are added to the end; new icons would be inserted throughout the grid in the proper alphabetical location, likely shifting everything around and creating a very large visual diff in the process.

We could just assume a bunch of new icons were added and approve the diff, though it would be difficult to verify the new icons looked ok in the toggled state given the visual noise of everything shifting around in the grid. Also, if the icon update included changes to existing icons alongside brand new icons, diffs for those changes to existing icons would be very difficult to resolve in the visual noise.

Given how seldomly icons are added (correct me if I'm wrong), I think it still makes sense to add a snapshot here, but I'm curious if anyone else has thoughts.